src/types/index.ts

```diff
@@ -0,0 +1,14 @@
+  
+/**
+ * @description NavButtonProps
```

Comment:
We don't typically use the @description tag, typically this would be 'Interface describing NavButtonProps'. And of course if based on the earlier comment you decide to move the NavButton into its own file/ component then this type can be in that file as Props.

If not, remove the other unnecessary JSDoc comments as described in previous comments

Reply:
done